doc/api/process.md

```diff
@@ -916,6 +916,63 @@ console.log(process.env.test);
 // => 1
 ```
 
+## process.envFlags
+<!-- YAML
+added: v10.0.0
```

Comment:
This should be `added: REPLACEME`. The releaser will substitute the actual version when making the release.

Reply:
I figured that wasn't right, but didn't know what to put.  Is it documented somewhere?

Reply:
@boneskull Heh, yes. :smile: I introduced these things, and I agree that this *should* be documented somewhere, but there’s one question that has always stopped me from doing it myself until now:

Where would people look for such documentation?

Reply:
CONTRIBUTING.md perhaps?

Reply:
If we could catch this on landing (i.e. with https://github.com/nodejs/node-core-utils/issues/193) then I think that would make a big difference, although documenting is a good idea too.